backends/bmv2/jsonconverter.cpp

```diff
@@ -967,6 +967,12 @@ JsonConverter::convertActionBody(const IR::Vector<IR::StatOrDecl>* body,
                         // Do not generate any code for this operation
                         continue;
                     }
+                } else if (em->originalExternType->name == v1model.directCounter.name) {
+                    if (em->method->name == v1model.directCounter.count.name) {
+                        BUG_CHECK(mc->arguments->size() == 0, "Expected 0 argument for %1%", mc);
+                        // Do not generate any code for this operation
```

Comment:
I don't understand this. If you generate no code for the count, then what is the point of having it?